src/Sylius/Component/Registry/spec/Fixture/AbstractSampleService.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace spec\Sylius\Component\Registry\Fixture;
+
+/**
+ * @author Beno!t POLASZEK
+ */
+abstract class AbstractSampleService
```

Comment:
We can remove this fixture class and use `\stdClass` instead :)

Reply:
Good catch, it doesn't even need to be abstract.